### ISSUE TITLE
[Accessibility] bibliography entries are not accessible in html output due to the deprecated "doc-biblioentry" role

### ISSUE THREAD

**Issue by jooyoungseo** [jgm/pandoc#8354]
Related to [quarto-cli#1841](https://github.com/quarto-dev/quarto-cli/issues/1841), I am reporting a critical accessibility issue found in pandoc bibliography html output. Tested with pandoc version 2.19.2.

Pandoc uses deprecated aria role "doc-biblioentry" that is not being recognized by some screen readers. This means bibliography entries are not correctly read by screen readers, such as JAWS.

## Reproducible Steps

`test.md`:

````
---
title: test
nocite: "[@*]"
references:
- abstract: The American National Academies of Sciences, Engineering,
    and Medicine (NASEM) has recently released the report
    "Reproducibility and Replicability in Science." The report has
    prompted discussions within many disciplines about the extent of the
    current adoption of reproducibility and replicability, the
    challenges involved in publishing reproducible results as well as
    strategies for improving. We organized a panel at the IEEE VIS
    conference 2019 to start a discussion on the reproducibility
    challenges faced by the visualization community and how those
    challenges might be addressed. In this viewpoint, we summarize key
    findings of the NASEM report, the panel discussion, and outline a
    set of recommendations for the visualization community.
  author:
  - family: Fekete
    given: Jean-Daniel
  - family: Freire
    given: Juliana
  citation-key: feketeExploringReproducibilityVisualization2020
  container-title: IEEE Computer Graphics and Applications
  DOI: 10.1109/MCG.2020.3006412
  id: feketeExploringReproducibilityVisualization2020
  ISSN: 1558-1756
  issue: 5
  issued: 2020-09
  page: 108-119
  source: IEEE Xplore
  title: Exploring Reproducibility in Visualization
  type: article-journal
  volume: 40
---

## References

````

``` bash
pandoc -s test.md -o test.html --citeproc
```

## JAWS Screen Reader Output

```
test
References
https://doi.org/10.1109/MCG.2020.3006412
```

* Note that only URL part is being recognized while other text are dropped.

## Solution

Take a lok at the html output code below:

``` html
<h1 class="title">test</h1>
  </header>
  <h2 class="unnumbered" id="references">References</h2>
  <div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
    <div id="ref-feketeExploringReproducibilityVisualization2020" class="csl-entry" role="doc-biblioentry">
      Fekete, Jean-Daniel, and Juliana Freire. 2020. <span>“Exploring
        Reproducibility in Visualization.”</span> <em>IEEE Computer Graphics and
        Applications</em> 40 (5): 108–19. <a
        href="https://doi.org/10.1109/MCG.2020.3006412">https://doi.org/10.1109/MCG.2020.3006412</a>.
    </div>
  </div>
```

1. Replace `role="doc-bibliography"` with `role="list"`

2. Replace `role="doc-biblioentry"` with `role="listitem"`

I have verified that this fix resolves the current accessibility issue. See also [example 2](http://kb.daisy.org/publishing/docs/html/dpub-aria/doc-biblioentry.html#ex).
